includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -195,4 +203,57 @@ private function isWikiaOrgCommunity() {
 
 		return wfGetBaseDomainForHost( parse_url( $wgServer, PHP_URL_HOST ) ) === $wgWikiaOrgBaseDomain;
 	}
+
+
+	/**
+	 * External API request to IntentX
+	 */
+	public function getFandomShopDataFromIntentX() {
+		global $wgCityId, $wgMemc, $wgFandomShopMap, $wgFandomShopUrl;
+
+		$key = self::MEMC_PREFIX_FANDOM_STORE;
+		// get id from parameter or default to city id
+		$id = intval( $this->getVal( static::PARAM_ID, $wgCityId) );
+		// not using wfMemcKey here to avoid prefix conflicts
```

Comment:
what kind of conflicts? wfMemcKey and other functions prevent memcache conflicts between wikis and environments (prod/preview/verify/production). Because otherwise your code executed on a sandbox can pollute the production cache.

Reply:
Because this is getting run on community, the prefix was being set to a wiki id rather than a domain name.

Reply:
using **wgSharedMemcKey**